src/initializers/koa/routes.ts

```diff
@@ -0,0 +1,23 @@
+import * as os from 'os';
+
+export default {
```

Comment:
I was thinking to just export the middleware. Not add them to the routes by the default. Don't have a strong opinion though. @anastasios-workable @nikosd23 wdyt?